doc/modules/tree.rst

```diff
@@ -572,6 +572,32 @@ Mean Absolute Error:
 
 Note that it fits much slower than the MSE criterion.
 
+.. _tree_missing_value_support:
+
+Missing Values Support
+======================
+
+:class:`~tree.DecisionTreeClassifier` and :class:`~tree.DecisionTreeRegressor`
+have built-in support for missing values when `splitter='best'` and criterion is
+`'gini'`, `'entropy`', or `'log_loss'`, for classification' or
+`squared_error'`, `'friedman_mse'`, or `'poisson'` for regression.
+
+For each potential threshold, the splitter will evaluate the split with all the
+missing values going to the left child or the right child. When predicting, the
+samples with missing values are assigned according::
+
+    >>> from sklearn.tree import DecisionTreeClassifier
+    >>> import numpy as np
+
+    >>> X = np.array([0, 1, 6, np.nan]).reshape(-1, 1)
+    >>> y = [0, 0, 1, 1]
+
+    >>> tree = DecisionTreeClassifier(random_state=0).fit(X, y)
+    >>> tree.predict(X)
+    array([0, 0, 1, 1])
+
+For a given feature, if no missing values are seen during training, then missing
+values are mapped to the child with the most samples.
```

Comment:
```suggestion
If no missing values are seen during training for a given feature, then during prediction
missing values are mapped to the child with the most samples.
```

Putting the "if no missing values" first because it is the most important thing about this sentence. Or at least the thing that lets you decide "is this the sentence I am looking for" when scanning the text.

WDYT?

Reply:
I agree.